index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Document</title>
+</head>
+<body>
+  <main id="main">
+    <h1 id="title">Alan Turing</h1>
+    <div id="img-div">
+      <caption></caption>
```

Comment:
- [x] Kindly remove any piece of unused code 🙏 . This will help declutter your webpage and improve loading speeds